apps/ledger-live-desktop/src/renderer/components/RequestAmount.tsx

```diff
@@ -1,10 +1,11 @@
-// @flow
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Is it intended ?

Reply:
Yes it is, I can remove the line and update the code to respect the linter. IMO this linter rule is kinda useless

Reply:
You should never remove completely an eslint rule from a file unless it being 100% necessary. 
Either update the code, if not possible use `eslint-disable-next-line` where needed, and only if repeated 10 times then it's worth discussing with the team about globally removing it from our eslint rules. 

In this specific example, it's purely styling related, so let's respect it for the sake of code unity.